File: official/docs/csharp/current/billing/create-stripe-bank-account-secret.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using EasyPost;
using Newtonsoft.Json;

namespace EasyPostExamples
{
    public class Examples
    {
        public static async Task Main()
        {
            var client = new EasyPost.Client(new EasyPost.ClientConfiguration("EASYPOST_API_KEY"));

            EasyPost.Models.API.StripeClientSecret response = await client.Beta.ReferralCustomer.CreateBankAccountClientSecret();

            Console.WriteLine(JsonConvert.SerializeObject(response, Formatting.Indented));
        }
    }
}

File: official/docs/java/current/billing/create-stripe-bank-account-secret.java
package billing;

import com.easypost.exception.EasyPostException;
import com.easypost.service.EasyPostClient;

public class CreateStripeBankAccountSecret {
    public static void main(String[] args) throws EasyPostException {
        EasyPostClient client = new EasyPostClient("EASYPOST_API_KEY");

        ClientSecret response = client.betaReferralCustomer.createBankAccountClientSecret();

        System.out.println(response);
    }
}
